playbooks/roles/edxapp/files/microsite_theming.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+# Copyright (c) Microsoft Corporation. All Rights Reserved.
+
+# Remove if edx-microsite folder exists
+if [[ -d /edx/app/edxapp/edx-microsite ]]; then
```

Comment:
nit: why double "[[" ? is one enough or are you following a pattern used elsewhere in oxa-tools . not blocking on this -- just a curiosity